domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/statementofmeans/BankAccount.java

```diff
@@ -45,6 +47,10 @@ public BankAccount(BankAccountType type, boolean joint, BigDecimal balance) {
         this.balance = balance;
     }
 
+    public String getTypeDescription() {
```

Comment:
Please remove, this is pointless. Can be easily achieved by `getType().getDescription()`

Reply:
As we are using PDF REST service and serialisation sends just name of enum and not the object. These getters are added to avoid different model in pdf content providers. where we used to convert enum into a string equivalent models. And we are using this property to display enum description in PDF template.